src/main/java/org/jitsi/jicofo/jigasi/JigasiDetector.java

```diff
@@ -41,6 +41,25 @@
     public static final String JIGASI_ROOM_PNAME
         = "org.jitsi.jicofo.jigasi.BREWERY";
 
+    /**
+     * The name of the stat used by the instance to indicate the number of
+     * participants. This should match
+     * {@code VideobridgeStatistics.NUMBEROFPARTICIPANTS}, but is defined
+     * separately to avoid depending on the {@code jitsi-videobridge}
+     * maven package.
+     */
+    private static final String STAT_NAME_PARTICIPANTS = "participants";
+
+    /**
+     * The name of the stat that indicates the instance has entered graceful
+     * shutdown mode.
+     * {@code VideobridgeStatistics.SHUTDOWN_IN_PROGRESS}, but is defined
+     * separately to avoid depending on the {@code jitsi-videobridge} maven
+     * package.
+     */
+    public static final String STAT_NAME_SHUTDOWN_IN_PROGRESS
```

Comment:
Can we have one copy of this constant in jicofo? There's one in JigasiDetector and one in Bridge.java